tools/ou3_p4_first_accel_replay_mismatch.py

```
@@ -0,0 +1,106 @@
+
+import ou3_p4_joint_first_accel_cover as C
+import ou3_p4_candidate_full_word as CAND
+import ou3_p4_joint_word_dissipation_design as D
+import ou3_p4_joint_word_gauge_design_v2 as G2
+import ou3_p4_source_node_cells as NODES
+
+DEFAULT_DOMAIN=C.DEFAULT_DOMAIN
+
+
+def _first_diff(a,b):
+    for i,(x,y) in enumerate(zip(a,b)):
```

Ruff (0.16.3)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)